src/Middleware/ExceptionMiddleware.php

```diff
@@ -264,16 +272,15 @@ protected function buildErrorLogData(ResponseInterface $response, \Throwable $e)
             'http_status' => $response->getStatusCode(),
             'env'         => $env,
             'message'     => $e->getMessage(),
-            'source'      => [
-                'file' => $e->getFile(),
-                'line' => $e->getLine(),
-            ],
+            'source'      => sprintf('%s:%s'. $e->getFile(), $e->getLine()),
             'trace'       => $e->getTrace(),
             'url'         => [
                 'raw'   => $currentUrl->__toString(),
                 'host'  => $currentUrl->getHost(),
                 'path'  => $currentUrl->getPath(),
-                'query' => $currentUrl->getAllQueryParams(),
+                'query' => $this->verbosity > self::VERBOSITY_MIN
```

Comment:
@kstupak lets make this a one-liner. Rule should be if its too long, as this case is, handle it by moving it up and assign it to a local var. Actually, we can move this to its own private method since we do use the exact logic for the referer as well, only the URL object changes.